docs/source/advanced/java-rxjava.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: 'Java and RxJava support'
+---
+
+import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
+
+
+Because Apollo Android is written in Kotlin, it can be used in Java projects. This guide will show you how to do that.
+
+## Use the Java codegen
+
+Apollo Android generates Kotlin code by default, but you can configure it to use the Java codegen instead:
+
+```kotlin:title=build.gradle[.kts]
+apollo {
+  generateKotlinModels.set(false)
```

Comment:
Feels weird to generate Java models but have samples in Kotlin below. What about splitting in 2 pages?

* java (contains only Java codeblocks): explains how to generate Java models and build an ApolloClient in Java and links to the rxjava page 
* rxjava (contains Java + Kotlin codeblocks): explains how to use RxJava wrappers in both Java and Kotlin

Reply:
That sounds good 👍  Will do.